### ISSUE TITLE
Alive?

### ISSUE THREAD

**Issue by marbetschar** [EmranAhmed/woo-product-download-from-amazon-s3#3]
Is this plugin still alive?

**Comment by EmranAhmed** [comment]
@marbetschar Yes, It is. But I am re-writing it. I will release it asap :) It will be great if you help me to fix some issues :)

Thank you

**Comment by marbetschar** [comment]
@EmranAhmed sounds great! Would be glad to help. Don't know which issues you exactly talking about - can only see two issues here on GitHub.

The thing on my end is, I need an S3 solution right now - the usual IT project thing, you know ;)

**Comment by marbetschar** [comment]
PS: How far is the rewrite away?

**Comment by trssssrt** [comment]
Following up, is the rewrite on its way?
